opacus/grad_sample/gsm_exp_weights.py

```diff
@@ -40,6 +43,13 @@ def __init__(
         )
 
     def forward(self, x: torch.Tensor, *args, **kwargs):
-        return self.call_for_per_sample_grads(
-            self._module, x.shape[0], x, *args, **kwargs
-        )
+        if torch.__version__ >= COMPATIBILITY_API_CUTOFF_VERSION:
+            return self.call_for_per_sample_grads(
+                module=self._module,
+                batch_size=x.shape[0],
+                loss_reduction=self.loss_reduction,
```

Comment:
Does this mean loss_reduction was not supported before? Should this be called out in our README too?